tools/webdriver/webdriver/__init__.py

```diff
@@ -36,3 +35,5 @@
     UnknownMethodException,
     UnsupportedOperationException,
     WebDriverException)
+from .bidi_session import (
```

Comment:
Having an `_` in module names is quite unusual in Python. Perhaps we could just have a name like `bidi`?

Reply:
Done.